spec/book_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './book'
+describe Book do
+  before :each do
+    @book = Book.new 'Title', 'Author'
+  end
```

Comment:
- [ ] Let ensure that there are unit tests for all the methods in all the classes, please kindly write unit test for the `add_rental` method in the Book class.